### ISSUE TITLE
Couldn't compile SFML/Cloth project

### ISSUE THREAD

**Issue by juliohq** [Marculonis21/CPPDrawing#1]
I tried the following:
```shell
cd SFML/Cloth
rm -rf build
mkdir build
cd build
cmake ..
make
```
And got this output:
```
[ 50%] Building CXX object CMakeFiles/Cloth.dir/src/main.cpp.o
In file included from /home/julio/CPPDrawing/SFML/Cloth/src/main.cpp:16:
/home/julio/CPPDrawing/SFML/Cloth/src/cloth.hpp:18:10: fatal error: 'omp.h' file not found
   18 | #include <omp.h>
      |          ^~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/Cloth.dir/build.make:79: CMakeFiles/Cloth.dir/src/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/Cloth.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

**Comment by Marculonis21** [comment]
Thank you @juliohq for the comment.

The include points to an OpenMP library which usually gets installed with the compiler. Based on your linux distro we could look for specific installation for you. For Ubuntu it would be `sudo apt install libomp-dev`. That should sort it out.  

Nevertheless, I've looked into the project and the Omp is kept there by accident as it is not used at all after that. I've removed it unnecessary includes to the file here a902313413bcb367fe1a03bf15dbecc5fbe03844 in *cloth.hpp* changes.

Hopefully it resolves your problem. Let me know!
